README.md

```
@@ -1,34 +1,455 @@
-| PostgreSQL Crash Course | https://youtu.be/SpfIwlAYaKk?feature=shared |
-| ReactJS in depth | https://youtube.com/playlist?list=PLHiZ4m8vCp9M6HVQv7a36cp8LKzyHIePr&si=m4ID_6isEhvPgWk4 |
-| CRUD API in Node JS and Express | https://www.youtube.com/watch?v=l8WPWK9mS5M&list=LL&index=60 |
+
+# UDDHAR - Disaster Management Platform
+
+![Uddhar Logo][logo]
+
+[![Contributors][contributors-shield]][contributors-url]
+
+
+<p align="center">
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: p

(MD033, no-inline-html)